Sources/Public/Animation/LottieView.swift

```diff
@@ -141,6 +141,16 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     }
   }
 
+  /// Returns a copy of this `LottieView` updated to have the specific configuration property
+  /// applied to its represented `LottieAnimationView` whenever it is updated via the `updateUIView(...)`
+  /// or `updateNSView(...)` methods.
+  public func configure<Property>(
+    _ property: ReferenceWritableKeyPath<LottieAnimationView, Property>,
+    value: Property
+  ) -> Self {
+    configure { $0[keyPath: property] = value }
```

Comment:
Could you add an overload with a `Property: Equatable` constraint where the value on the `LottieAnimationView` is only updated when the value actually changes?

Reply:
Sounds good 💪